src/fiber/ReactHardwareFiberComponent.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
I’m keeping them separated for now since there is a good chance I’ll want to change from POJOs to custom class instances. Keeping them separate is still nice for general code organization.